engines/director/detection_tables.h

```diff
@@ -5861,6 +5862,10 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINDEMO1("wwanimals", "Demo", "ANIMALS.EXE", "25fee176193d52cf7cbc2b604bfcbf8c", 6961631, 404),
 
+	WINGAME1_l("www", "", "WWW.EXE", "t:ae93455c01b997db10ed6cab146eda0d", 702365, Common::DE_DEU, 404),
+	// Original filename is "Wellen, Wracks und Wassermänner"
+	MACGAME1_l("www", "", "xn--Wellen, Wracks und Wassermnner-ltc", "t:c403ef687d52803dfae36077159674b8", 1103744, Common::DE_DEU, 404),
+
```

Comment:
Minor remarks:
1. The MAC entry should be above the WIN entry
2. WWW.EXE is a common name, it would be preferable to add another file to the detection entry
3. The checksum for the MAC entry should be r:<checksum> or tr:<checksum> if you want to use the tail

Reply:
Thx for the review, I force-pushed a fixed version.

Reply:
I need to change the hashes again, please don't merge yet.

Reply:
Now the hashes should be fine, its ready for a re-review.